R/api.R

```diff
@@ -23,6 +23,7 @@ build_api <- function(runner, path, backup_period = NULL,
   api$handle(endpoint_workflow_summary(runner))
   api$handle(endpoint_workflow_run(runner))
   api$handle(endpoint_workflow_status(runner))
+  api$handle(endpoint_report_versions(path))
```

Comment:
this endpoint wasn't actually wired up! not caught because missing integration tests :woman_facepalming:

Reply:
Oops, good spot